### DISCUSSION TITLE
"Assault enemy" order on abandoned/enemy outposts is broken

### DISCUSSION THREAD

**Discussion by JustOneRandomPlayer** [FakeFishGames/Barotrauma#14180] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When bots try to attack with an "Assault Enemy" order, they reach the exit of the sub and ignore it (before Summer Update they canceled it). 
If except the "Assault Enemy" order, they have something else forcing them to return (turret, reactor, etc.), then they will cyclically get stuck in these orders, forever trying to attack and returning. But in the case of turrets, bots may also begin to ignore the "Assault Enemy" order when they start use the turret.

### Reproduction steps

1. Start "abandoned outpost" mission, or dock at an enemy faction's outpost.
2. Order the armed bot to "Assault Enemy".
3. Bot will reach the exit from the submarine and will ignore the order, or will cyclically get stuck with another order.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.8.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested the fix for this.
Closing discussion as resolved.
